mk/cfg/x86_64-unknown-linux-uclibc.mk

```diff
@@ -0,0 +1,29 @@
+# x86_64-unknown-linux-uclibc configuration
+CC_x86_64-unknown-linux-uclibc=$(CFG_UCLIBC_ROOT)/bin/uclibc-gcc
```

Comment:
Probably. FWIW, the convention is to set these variables to `x86_64-linux-uclibc-gcc` (no `unknown` component) and the user is supposed to make `x86_64-linux-uclibc-gcc` somehow (usually by symlinks somewhere in PATH).